explainer-server/app/actions.scala

```diff
@@ -24,12 +24,13 @@ trait AuthActions {
     cookie <- req.cookies.get(publicSettingsService.publicSettings.assymCookieName)
   } yield PanDomain.authStatus(cookie.value, PublicKey(publicKey))
   
-  object PandaAuthenticated extends AuthenticatedBuilder(req => userAuthStatusOptFor(req) match {
+  class PandaAuthenticated @Inject() (config: Config) extends AuthenticatedBuilder(req => userAuthStatusOptFor(req) match {
```

Comment:
Why not compile time dependency injection? see: https://github.com/guardian/recommendations/pull/19